src/NetscriptFunctions/Go.ts

```diff
@@ -41,17 +41,17 @@ export function NetscriptGo(): InternalAPI<NSGo> {
   return {
     makeMove:
       (ctx: NetscriptContext) =>
-      (_x, _y): Promise<Play> => {
+      async (_x, _y): Promise<Play> => {
```

Comment:
None of these should be async either.

Reply:
weaken is async in this way https://github.com/bitburner-official/bitburner-src/blob/dev/src/NetscriptFunctions.ts#L353 m but not hack or grow for some reason

Reply:
weaken probably shouldn't be either. *Or* it could be rewritten to use `await`, which would flow a bit better.